test/ci-op-configs-mirror-integration/run.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+
+WORKDIR="$( mktemp -d )"
+trap 'rm -rf "${WORKDIR}"' EXIT
+
+TEST_ROOT="$( dirname "${BASH_SOURCE[0]}")"
+INPUT_FILES="${TEST_ROOT}/data/input"
+EXPECTED="${TEST_ROOT}/expected.yaml"
+
+echo "[INFO] Running ci-op-configs-mirror in dry-mode..."
+if ! ci-op-configs-mirror --to-org super-priv --config-path "${INPUT_FILES}" --dry-run --private 2> "${WORKDIR}/ci-op-configs-mirror-stderr.log" > "${WORKDIR}/ci-op-configs-mirror-stdout.log"; then
```

Comment:
I'm not a great fan of this special output for testing. This tool does not interact with anything external, it's just producing files, so it does not need anything like `--dry-run` at all. The integration tests can let it generate the files and then compare the full tree with `diff -Naupr`, just like [repo-init](https://github.com/openshift/ci-tools/blob/master/test/repo-init-integration/run.sh#L77) or [prowgen](https://github.com/openshift/ci-tools/blob/master/test/prowgen-integration/run.sh#L27) tests are doing it.

Reply:
I am ok with generating the files. I will address this.